pkg/virt-handler/vm.go

```diff
@@ -1352,6 +1383,56 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 	}
 
 	if d.isPreMigrationTarget(vmi) {
+		for i, volume := range vmi.Spec.Volumes {
```

Comment:
can we break this out into a helper function?

Reply:
Done